webapp/client/src/pages/TaxNumberPage.js

```diff
@@ -0,0 +1,331 @@
+        case TAX_NUMBER_FORM_STATES.TaxOfficeSelection:
+        case TAX_NUMBER_FORM_STATES.RequestNewTaxNumber:
+          setTaxNumberFormState(TAX_NUMBER_FORM_STATES.TaxOfficeSelection);
+          break;
+        default:
+          break;
+      }
+    } else if (taxNumberFormState === TAX_NUMBER_FORM_STATES.TaxNumberInput) {
+      setTaxNumberFormState(
+        TAX_NUMBER_FORM_STATES.StateSelectionTaxNumberExists
+      );
+    } else {
+      setTaxNumberFormState(
+        TAX_NUMBER_FORM_STATES.StateSelectionNoTaxNumberExists
+      );
+    }
+  };
+
+  const changeTaxOffices = (event) => {
+    if (
+      event.target.value &&
+      taxNumberFormState === TAX_NUMBER_FORM_STATES.TaxOfficeSelection
+    ) {
+      setTaxNumberFormState(TAX_NUMBER_FORM_STATES.RequestNewTaxNumber);
+    } else if (
+      !event.target.value &&
+      taxNumberFormState === TAX_NUMBER_FORM_STATES.RequestNewTaxNumber
+    ) {
+      setTaxNumberFormState(TAX_NUMBER_FORM_STATES.TaxOfficeSelection);
+    }
+  };
+
+  const stateDropDown = (
+    <FormFieldDropDown
+      fieldName="bundesland"
+      fieldId="bundesland"
+      key="bundesland"
+      selectedValue={fields.bundesland.selectedValue}
+      options={fields.bundesland.options}
+      label={{
+        text: t("lotseFlow.taxNumber.bundesland.labelText"),
+      }}
+      errors={fields.bundesland.errors}
+      onChangeHandler={changeStateSelection}
+      defaultValue="Bitte auswählen"
+    />
+  );
+
+  const bufaNrDropdown = (
+    <FormFieldDropDown
+      fieldName="bufa_nr"
+      fieldId="bufa_nr"
+      key="bufa_nr"
+      selectedValue={fields.bufaNr.selectedValue}
+      options={extractCorrespondingTaxOffices(
+        selectedStateAbbreviation,
+        taxOfficeList
+      )}
+      label={{
+        text: t("lotseFlow.taxNumber.taxOffices.labelText"),
+      }}
+      errors={fields.bufaNr.errors}
+      onChangeHandler={changeTaxOffices}
+      defaultValue="Bitte auswählen"
+    />
+  );
+
+  const taxNumberInput = (
+    <FormRowCentered key="steuernummerRow">
+      <FormFieldTaxNumber
+        required
+        fieldName="steuernummer"
+        fieldId="steuernummer"
+        values={fields.steuernummer.value}
+        label={{
+          text: t("lotseFlow.taxNumber.taxNumberInput.labelText"),
+          exampleInput: t("lotseFlow.taxNumber.taxNumberInput.labelText"),
+        }}
+        errors={fields.steuernummer.errors}
+        key={selectedStateAbbreviation} // Added render a new DOM Element whenever the selected state changes
+        isSplit={selectedStateAbbreviation !== "he"} // Do not split field for Hessen
+      />
+    </FormRowCentered>
+  );
+
+  const requestNewTaxNumberCheckbox = (
+    <FormRowCentered key="requestNewTaxNumberRow">
+      <SubHeading>
+        {t("lotseFlow.taxNumber.requestNewTaxNumber.headline")}
+      </SubHeading>
+      <div>{t("lotseFlow.taxNumber.requestNewTaxNumber.intro")}</div>
```

Comment:
Is there a specific reason you're making this a div not a paragraph?

Reply:
I kept it as it was.

Reply:
But why not